kotest-extensions/kotest-extensions-testcontainers/src/jvmMain/kotlin/io/kotest/extensions/testcontainers/StartablePerSpecListener.kt

```diff
@@ -38,6 +39,28 @@ class StartablePerSpecListener<T : Startable>(val startable: T) : TestListener {
       }
    }
 
+   override suspend fun beforeContainer(testCase: TestCase) {
+      if (testCase.type != TestType.Container) return
+
+      withContext(Dispatchers.IO) {
+         testLifecycleAwareListener.beforeTest(testCase)
+      }
+   }
+
+   override suspend fun beforeEach(testCase: TestCase) {
+      if (testCase.type != TestType.Test) return
+
+      withContext(Dispatchers.IO) {
+         testLifecycleAwareListener.beforeTest(testCase)
```

Comment:
These shouldn't be implemented as the StartablePerSpecListener doesn't reset on each test.